<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>166</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввести последовательность</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>230</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Агоритмические</string>
    </property>
    <widget class="QTableWidget" name="tb1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>350</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>62</number>
     </attribute>
     <column>
      <property name="text">
       <string>0..9</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>10..99</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>100..999</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="lbl_alg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Результат: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>230</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Табличные</string>
    </property>
    <widget class="QTableWidget" name="tb2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>350</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>62</number>
     </attribute>
     <column>
      <property name="text">
       <string>0..9</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>10..99</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>100..999</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="lbl_tbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Результат: </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_seq">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>463</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
